src/quacc/schemas/ase.py

```diff
@@ -271,6 +271,40 @@ def summarize_vib_and_thermo(
     )
 
 
+def summarize_neb_run(dyn: Optimizer) -> OptSchema:
+    """
+    Summarizes the NEB run results into an OptSchema.
+
+    Parameters
+    ----------
+    dyn : Optimizer
+        The ASE Optimizer object used in the NEB run.
+
+    Returns
+    -------
+    OptSchema
+        A dictionary containing the optimization summary, including parameters,
+        trajectory, and trajectory results.
+    """
+    # Get trajectory
+    trajectory = (
+        dyn.traj_atoms
+        if hasattr(dyn, "traj_atoms")
+        else read(dyn.trajectory.filename, index=":")
+    )
```

Comment:
This logic has since been updated in `summarize_opt_run`. There is no `traj_atoms` attribute anymore. Please update to match what's being done in `summarize_opt_run`.